action.yaml

```diff
@@ -346,6 +346,20 @@ runs:
     shell: bash
     run: rm utils.R
 
+  - name: Set repositories
+    run: |
+      cat > ${HOME}/.Rprofile <<EOF
+      options(
+        repos = c(
+          RSPM = "https://packagemanager.posit.co/cran/__linux__/jammy/latest",
+          CRAN = "https://packagemanager.posit.co/cran/__linux__/jammy/latest"
+        )
+      )
+      EOF
```

Comment:
I added `| tail -1` to this, because for example on my local machine `lsb_release --codename --short` returns:
```
No LSB modules are available.
bookworm
```